anvill/python/anvill/binja/bnfunction.py

```diff
@@ -152,6 +163,9 @@ def _extract_types(
         rdi, rsi as operands in the MLIL, we should check if they have pointer information)
         """
         results = []
+        if not (item_or_list and str(item_or_list)):
```

Comment:
What is going on here with the `str`?

Reply:
`item_or_list` may not be an empty object but it's string representation may be empty. In that case there is possibility that register name is empty for unimplemented operands.

Reply:
Huh, can you add a comment for this. That is rather subtle.